### ISSUE TITLE
Sometimes Chain-Repeat Animation do not work as I expect

### ISSUE THREAD

**Issue by pangyulei** [icanzilb/EasyAnimation#2]
```
UIView.animateAndChainWithDuration(1.0, delay: 0.0, options: nil, animations: { () -> Void in
    self.layer.borderColor = flashBorderColor.CGColor
}, completion: nil).animateWithDuration(1.0, delay: 0.0, options: UIViewAnimationOptions.Repeat, animations: { () -> Void in
    self.layer.borderColor = originBorderCGColor
}, completion: nil)
```
I expect it could flash borderColor, but not works for me.
already read the demoApp's example, can not figure out what's going wrong here, pls help :(

**Comment by icanzilb** [comment]
I'll try to reproduce later today

**Comment by icanzilb** [comment]
this might be a bug related to animating only layer properties in the animations block; definitely high prio on my todo list, will let you know when I upload a fix

**Comment by icanzilb** [comment]
@pangyulei "but not works for me." - what exactly happens?

**Comment by pangyulei** [comment]
@icanzilb I set NSTimeInterval to 1 sec, but it change borderColor immediately.

**Comment by icanzilb** [comment]
okay so it just flashes very quickly between the two colors, right?

**Comment by pangyulei** [comment]
@icanzilb right

**Comment by icanzilb** [comment]
alright - that's a bug I'm working currently on I'll release a new pod version by end of the week to publish the fix

**Comment by icanzilb** [comment]
fixed in the latest version 0.7.0 - give it a try please and let me know

**Comment by pangyulei** [comment]
@icanzilb works fine now! thx!
